src/EditorFeatures/Core/Options/LegacyGlobalOptionsWorkspaceService.cs

```diff
@@ -23,22 +23,22 @@ internal sealed class LegacyGlobalOptionsWorkspaceService : ILegacyGlobalOptions
         private readonly CodeActionOptionsStorage.Provider _provider;
 
         private static readonly Option2<bool> s_generateOverridesOption = new(
-            "GenerateOverridesOptions_SelectAll", defaultValue: true);
+            "dotnet_select_all_when_generate_overrides", defaultValue: true);
```

Comment:
>dotnet_select_all_when_generate_overrides [](http://example.com/codeflow?start=13&length=41)

Maybe `dotnet_generate_all_overrides`? But not sure what this means exactly.

Reply:
If I remembered correctly this is a default value for the checkbox in popup window

Reply:
I feel it might be `dotnet_generate_all_overrides` or `dotnet_generate_overrides_for_all_members`
I would use the later one, but keep this comment open if you have any other suggestions.